### DISCUSSION TITLE
Navigating a PDF of slides

### DISCUSSION THREAD

**Discussion by hasantahir** [ahrm/sioyek#249] [Ideas]
Is it possible to navigate a PDF containing slides page by page? At the moment, the `Pg Up` and `Pg Down` offset shifts about half a page. On a similar note, is it possible to just display a single page of the PDF on screen? While presenting or teaching, such a feature will come in handy.

Thanks for the great work. It has now become my default PDF viewer.

**Comment by ahrm** [comment]
We have a `toggle_presentation_mode` command that does that.

**Comment by hasantahir** [comment]
Fantastic, thanks.
